imperial_coldfront_plugin/templates/imperial_coldfront_plugin/user_search.html

```diff
@@ -0,0 +1,33 @@
+{% extends "common/base.html" %}
+{% block content %}
+  <form method="post">
+    {% csrf_token %} {{ form.as_p }}
+    <input type="submit" value="Search" />
+  </form>
+  {% if search_results %}
+    <form action="{% url 'imperial_coldfront_plugin:send_group_invite' %}" method="post"">
+      {% csrf_token %}
+      <table>
+        <tr>
+          <th>Name</th>
+          <th>Email</th>
+          <th>Username</th>
+          <th>Department</th>
+          <th></th>
```

Comment:
```suggestion
          <th>Select</th>
```

? Subjective, but it took me a quick second to see the radio button at first.

Reply:
Very valid observation I'll leave it for the time being as the whole appearance needs a general pass.